syntaxes/diagrams/erDiagram.yaml

```diff
@@ -42,12 +42,12 @@
           name: keyword.control.mermaid
     - comment: '(entity) (relationship) (entity) : (label)'
       match: !regex |-
-        \s*([\w-]+)\s+ # entity
+        \s*([\w-]+)\s() # entity
```

Comment:
Should this be a capture group or should it be a `*` similar to the other changes?

Reply:
Oops, fixed!